include/recorders.h

```diff
@@ -53,16 +53,21 @@ struct task_record {
 
 class task_recorder {
   public:
-	using task_record = std::vector<task_record>;
+	using task_records = std::vector<task_record>;
```

Comment:
Why the renaming? A "record" could be considered a single instance or a collection of instances (which I assume was the intent, cc @PeterTh). In any case, it's inconsistent with ` command_recorder::command_record` now!